spark/src/main/kotlin/com/adevinta/spark/components/chips/InputChip.kt

```diff
@@ -111,6 +90,14 @@ import androidx.compose.material3.InputChip as MaterialInputChip
  * for this chip. You can create and pass in your own `remember`ed instance to observe
  * [Interaction]s and customize the appearance / behavior of this chip in different states.
  */
+@Deprecated(
+    message = "Use one of the options: ChipOutlined, ChipFilled, ChipTinted, ChipDashed",
```

Comment:
Material Input Chip is closer to Outlined than the other styles variant so we should recommend this one only instead